test/image/mocks/zz02-carpet_grid_dash.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Please rename this file to `z-carpet_grid_dash.json`. There should be no need for `zz02`.